packages/flutter_tools/lib/src/debug_adapters/flutter_adapter.dart

```diff
@@ -83,11 +83,41 @@ class FlutterDebugAdapter extends DartDebugAdapter<FlutterLaunchRequestArguments
   @override
   bool get terminateOnVmServiceClose => false;
 
+  /// Whether or not the user requested to debug (via noDebug).
+  ///
+  /// This field is always `false` until a launchRequest or attachRequest has been
+  /// received and is then set according to the noDebug value.
+  ///
+  /// If `false`, we will not connect to the VM Service and some functionality
+  /// will not be available. Functionality provided via the daemon (hot reload
+  /// etc.) will still be available.
+  ///
+  /// debug/noDebug here refers to the DAP "debug" mode and not the Flutter
+    // debug mode (vs Profile/Release). It is possible for the user to "Run"
```

Comment:
looks like the formatting of the comments on lines 96,98 got messed up, no longer dartdocs plus an extra indent.

Reply:
Oops, fixed!